hubblestack_nova/cve_scan_v2.py

```diff
@@ -114,6 +129,15 @@ def audit(data_list, tags, debug=False, **kwargs):
 
             ttl = data['cve_scan_v2']['ttl']
             url = data['cve_scan_v2']['url']
+
+            # get whitelist from pillar data if exists
+            whitelist = None
+            cve_scan_v2_pillar = __pillar__.get('cve_scan_v2', {})
```

Comment:
Can you please switch this to use `config.get` instead of `__pillar__.get`? That will make it more flexible and allow us to use it for hubble in addition to hubble-salt.

[example](https://github.com/hubblestack/hubble-salt/blob/develop/_returners/splunk_nova_return.py#L271)